region/maridia/outer/Mama Turtle Room.json

```diff
@@ -717,18 +733,6 @@
         "With a fail, if you haven't moved horizontally, simply fall straight down to land safely back on Mama Turtle."
       ]
     },
-    {
-      "id": 32,
-      "link": [4, 3],
-      "name": "Base",
-      "requires": [
-        {"or": [
-          "canWalljump",
-          "SpaceJump",
-          "HiJump"
```

Comment:
I think hijump can remain here, then have hijump + careful for [4, 6]

Reply:
I just moved this whole thing 4->6 without editing; I'm 99% sure this existing strat was to the ledge, because otherwise you have no way to break the shot block.

Reply:
you can shoot down to grab the item if you miss the ledge.

Reply:
That's strat 33 tho?

Reply:
33 is shooting the block and jumping across. It requires timing the shots and avoiding having Mama T take the hits. With HiJump you can just jump over and then shoot down - which is what this strat was including.

Reply:
5afdc5ba

Reply:
Got it, I understand now!